src/ToDoList.js

```diff
@@ -22,13 +22,12 @@ const containerStyle = css`
   align-items: center;
   flex-direction: column;
   gap: 30px;
-`;
-
-const titleStyle = css`
-  color: #29b6f6;
-  font-size: 40px;
-  font-weight: bold;
-  margin-top: 30px;
+  h1 {
+    color: #29b6f6;
+    font-size: 50px;
+    font-weight: bold;
+    margin-top: 30px;
```

Comment:
I don't think it's needed.
just increase the gap

Reply:
what?